plugins/pubsub/mocks/build.bzl

```diff
@@ -0,0 +1,27 @@
+def _generate_manifest(context):
```

Comment:
should we move this into [rules_player](https://github.com/player-ui/rules_player) so we dont duplicate this file in multiple packages?